pandas/tests/window/test_groupby.py

```diff
@@ -732,6 +732,42 @@ def test_groupby_level(self):
         )
         tm.assert_series_equal(result, expected)
 
+    def test_as_index_false(self):
```

Comment:
can you add the multi-key groupby here as well (parameterize if you can)

```
df_res2 = df.groupby([df.id, df.index.weekday], as_index=False).rolling(window=2, min_periods=1).mean()
df_res3 = df.groupby([df.id]).rolling(window=2, min_periods=1).mean()
df_res4 = df.groupby([df.id], as_index=False).rolling(window=2, min_periods=1).mean()
```
e.g. 2 & 4 (we likley have 1 & 3 covered, but wouldn't object to those included as well)

Reply:
Hmm all these tested (except the `as_index=True` cases which are tested everywhere else). So you just want it parameterized?

Reply:
if they are elsewhere then don't (just the cases that is covering in this PR are fine). parameterize if you can.